README.md

```diff
@@ -13,43 +16,196 @@ This project is still heavily under development and is not ready for use
 **yet**. However, if you want to experiment, please do try running the current
 development build and reporting any issues you run into.
 
-## Requirements
+## Quickstart
+
+> Prebuilt images for cert-manager are made available on Dockerhub.
+
+This guide sets up cert-manager to run as a Deployment on your Kubernetes
+cluster. It will then go on to set up the Letsencrypt staging server as a
+Certificate issuer, and request a Certificate for a domain you control using
+the HTTP01 challenge mechanism.
+
+This guide will configure cert-manager to attempt to fulfill Certificate
+resources in all namespace in your cluster.
+
+### 0. Pre-requisites
 
 * Kubernetes cluster with CustomResourceDefinitions enabled (1.7+) (see [#49](https://github.com/jetstack-experimental/cert-manager/issues/49))
 
-## Deploying
+### 1. Deploy cert-manager
 
-To deploy the latest development version, run:
+To deploy the latest version of cert-manager, run:
 
 ```
-$ kubectl create -f docs/cert-manager.yaml
+$ kubectl create -f https://github.com/jetstack-experimental/cert-manager/blob/master/docs/cert-manager.yaml
 ```
 
-## Getting started
+In future this may be replaced with a Helm chart. There are currently no
+official RBAC roles defined for cert-manager (see [#34](https://github.com/jetstack-experimental/cert-manager/issues/34))
```

Comment:
❓ These look like they should be separate bullet points. And highlighted as "Note" maybe?